gradle/artifacts/publishAndroidNativePublications.txt

```diff
@@ -1,1320 +1,800 @@
-io.ktor:ktor-call-id-androidnativearm32/.jar
 io.ktor:ktor-call-id-androidnativearm32/.klib
 io.ktor:ktor-call-id-androidnativearm32/javadoc.jar
-io.ktor:ktor-call-id-androidnativearm32/kdoc.jar
 io.ktor:ktor-call-id-androidnativearm32/sources.jar
-io.ktor:ktor-call-id-androidnativearm64/.jar
 io.ktor:ktor-call-id-androidnativearm64/.klib
 io.ktor:ktor-call-id-androidnativearm64/javadoc.jar
-io.ktor:ktor-call-id-androidnativearm64/kdoc.jar
 io.ktor:ktor-call-id-androidnativearm64/sources.jar
-io.ktor:ktor-call-id-androidnativex64/.jar
 io.ktor:ktor-call-id-androidnativex64/.klib
 io.ktor:ktor-call-id-androidnativex64/javadoc.jar
-io.ktor:ktor-call-id-androidnativex64/kdoc.jar
```

Comment:
`.jar` and `kdoc.jar` were empty JARs so it should be okay to remove them from publishing.
@e5l do you know why we had been publishing these artifacts for native targets?